executor/src/lib.rs

```diff
@@ -218,6 +223,40 @@ impl ProcessedVMOutput {
     pub fn executed_trees(&self) -> &ExecutedTrees {
         &self.executed_trees
     }
+
+    pub fn accu_root(&self) -> HashValue {
```

Comment:
can we rename it just to be `root_hash`?

Reply:
Would this be confusing since we have both trees inside. state_root_hash and accu_root_hash.

Reply:
state_id?

Reply:
There is another method `state_root` after this line... would that still be confusing?